index.html

```diff
@@ -0,0 +1,255 @@
+          <img src="images/like.png" class="solutions__img" />
+          <p class="solutions__title">High security to protect from piracy</p>
+          <p class="solutions__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</p>
+        </div>
+        <div class="solutions__item">
+          <img src="images/headset.png" class="solutions__img" />
+          <p class="solutions__title">High security to protect from piracy</p>
+          <p class="solutions__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</p>
+        </div>
+    </div>
+    <section class="more">
+      <div class="more__item more__item--left">
+        <p class="small-title text-indigo-500  py-10 text-2xl font-bold">Why should you work with us?</p>
+        <p class="title">To upscale your business to the next level</p>
+        <p class="more__desc more__desc--small">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua invidunt ut labore.</p>
+      </div>
+      <div class="more__item more__item--right">
+        <div class="more__details">
+          <span class="more__icon" ><img src="images/arow.png"/></span>
+          <p class="more__desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.</p>
+        </div>
+        <div class="more__details">
+          <span class="more__icon" ><img src="images/arow.png"/></span>
+          <p class="more__desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.</p>
+        </div>
+        <div class="more__details">
+          <span class="more__icon" ><img src="images/arow.png"/></span>
+          <p class="more__desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.</p>
+        </div>
+      </div>
+     </section>
+
+     <section class="chart">
+      <div class="chart__item chart__item--left">
+        <p class="title">more impressions, more conversions</p>
+        <p class="chart-info__desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua invidunt ut labore.</p>
+        <button class="chart__button">
+          Get Free trial
+        </button>      </div>
+      <div class="more__item more__item--right">
+        <img src="images/chart.png" class="" />
+      </div>
+     </section>
+
+ <section class="logos">
+    <img  src="images/img1.PNG" class="logos__item" />
+    <img  src="images/img2.PNG"  class="logos__item" />
+    <img  src="images/img3.PNG"  class="logos__item" />
+    <img  src="images/img4.PNG"  class="logos__item" />
+    <img  src="images/img5.PNG"  class="logos__item" />
+ </section>
+ <section class="sales">
+    <div class="sales__item sales__item--left">
+      <p class="small-title text-indigo-500 py-6 font-bold text-2xl">Discover More</p>
```

Comment:
_smal-title_ is not a good name for a block .
 you could have used like a _text--small_ because you may need to re-use this somewhere else.